### PR TITLE
Increase queues limits

### PR DESCRIPTION
**Pull request by mtfranchetto**
## Changes

- increases queues limits to 10 and compute resources to 5
- fixes a bug introduced after #67 

## How Has This Been Tested?

Validated with a dry run after adding 10 queues and 5 compute resources on the first queue

## PR Quality Checklist

- [x] I added tests to new or existing code
- [ ] I removed hardcoded strings and used [`react-i18next`](https://react.i18next.com/) library ([useTranslation hook](https://react.i18next.com/latest/usetranslation-hook) and/or [Trans component](https://react.i18next.com/latest/trans-component)), see an example [here](https://github.com/aws/aws-parallelcluster-ui/commit/a6f1e2aa46b245b5bf7500a04b83195477a5cfa5)
- [ ] I made sure no sensitive info gets logged at any time in the codebase (see [here](https://cheatsheetseries.owasp.org/cheatsheets/Logging_Cheat_Sheet.html)) (e.g. no user info or details, no stacktraces, etc.)
- [ ] I checked that infrastructure/update_infrastructure.sh runs without any error
- [ ] I checked that `npm run build` builds without any error
- [ ] I checked that clusters are listed correctly
- [x] I checked that a new cluster can be created (config is produced and dry run passes)
- [ ] I checked that login and logout work as expected

In order to increase the likelihood of your contribution being accepted, please make sure you have read both the [Contributing Guidelines](../CONTRIBUTING.md) and the [Project Guidelines](../PROJECT_GUIDELINES.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: frontend/locales/en/strings.json [hunk 1]
```diff
@@ -798,7 +798,7 @@
       },
       "container": {
         "title": "Queues",
-        "description": "Configure up to 5 queues (Slurm partitions)."
+        "description": "Configure up to 10 queues (Slurm partitions)."
```
Comment (mendaomn) [inline frontend/locales/en/strings.json:801]: suggestion(non-blocking): does it make sense to parametrize the limit here?

File: frontend/src/old-pages/Configure/Queues/Queues.test.tsx [hunk 2]
```diff
@@ -49,11 +59,151 @@ describe('Given a list of instances', () => {
...
+        </MockProviders>,
+      )
+
+      const button = getByText('Add queue')
+      fireEvent.click(button)
+      expect(setState).not.toHaveBeenCalled()
+    })
+  })
+
+  describe("when they're 10 or less", () => {
```
Comment (mendaomn) [inline frontend/src/old-pages/Configure/Queues/Queues.test.tsx:106]: nit-pick: if I understand correctly, if they're 10 the button is disabled

File: frontend/src/old-pages/Configure/Queues/Queues.test.tsx [hunk 3]
```diff
@@ -49,11 +59,151 @@ describe('Given a list of instances', () => {
...
+      const button = getByText('Add compute resource')
+      fireEvent.click(button)
+      expect(setState).not.toHaveBeenCalledWith(
+        ['app', 'wizard', 'config', 'Scheduling', 'SlurmQueues', 0],
+        expect.anything(),
+      )
+    })
+  })
+
+  describe('when the compute resources of a queue are 5 or less', () => {
```
Comment (mendaomn) [inline frontend/src/old-pages/Configure/Queues/Queues.test.tsx:173]: nit-pick: if I understand correctly, if they're 5 the button is disabled

File: frontend/src/old-pages/Configure/Queues/Queues.test.tsx (hunk 2 above)
Comment (mtfranchetto) [reply frontend/src/old-pages/Configure/Queues/Queues.test.tsx:106]: nope, 10 queues are allowed!

File: frontend/src/old-pages/Configure/Queues/Queues.test.tsx (hunk 3 above)
Comment (mtfranchetto) [reply frontend/src/old-pages/Configure/Queues/Queues.test.tsx:173]: nope, 5 compute resources are allowed!

**Review by mendaomn — APPROVED** [review]

**Pull request merged**